bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -238,15 +238,27 @@ public Integer getSample() {
                     return pendingAddOps.size();
                 }
             });
-        initializeExplicitLacFlushPolicy();
+
+        initializeWriteHandleState();
+    }
+
+    protected void initializeWriteHandleState() {
+        if (!getLedgerMetadata().isClosed()
```

Comment:
Is this possible? Do we have a test case?

Reply:
It shouldn't be possible. in a write handle. In a read handle, it is.